tests/BenchmarkDotNet.IntegrationTests/BenchmarkDotNet.IntegrationTests.csproj

```diff
@@ -30,7 +30,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
```

Comment:
You need to update the `NuGetReferenceTests` integration tests implementation, as it uses this library to test NuGet related BDN feature:

```log
BenchmarkDotNet.IntegrationTests.NuGetReferenceTests.UserCanSpecifyCustomNuGetPackageDependency
The following benchmarks have failed to build: WithCallToNewtonsoft.SerializeAnonymousObject: Dry(NuGetReferences=Newtonsoft.Json 11.0.2, Toolchain=.NET Framework 4.8, IterationCount=1, LaunchCount=1, RunStrategy=ColdStart, UnrollFactor=1, WarmupCount=1)
Expected: True
Actual:   False
```

Reply:
The test fails because only a higher version can be used. I tried to suppress the error, but the internal project then crashes with another error.

It's hard to print good validation error, because we need to analyze `csproj` which almost impossible.

I downgraded to the first non-vulnerable version (`13.0.1`) and added package `13.0.2` in this test.